backend/tests/apps/owasp/graphql/nodes/project_health_stats_test.py

```
@@ -0,0 +1,78 @@
+            "projects_percentage_unhealthy",
+            "total_contributors",
+            "total_forks",
+            "total_stars",
+        }
+        assert expected_field_names == field_names
+
+    def _get_field_by_name(self, name):
+        return next(
+            (
+                field
+                for field in ProjectHealthStatsNode.__strawberry_definition__.fields
```

Pylint (3.3.7)

[error] Class 'ProjectHealthStatsNode' has no '__strawberry_definition__' member

(E1101)